From: Julius Bairaktaris <julius@bairaktaris.de>
Subject: [PATCH] nss-drv-Fix-nss_wifili_if-compilation-error-GCC-13

Imported from https://github.com/kuncy7/nss-packages qca-nss-drv/patches/0007-nss-drv-Fix-nss_wifili_if-compilation-error-GCC-13.patch (added in d7b3110); the patch file carries no author or description.
---
nss-drv-Fix-nss_wifili_if-compilation-error-GCC-13

Signed-off-by: Julius Bairaktaris <julius@bairaktaris.de>

--- a/exports/nss_wifili_if.h
+++ b/exports/nss_wifili_if.h
@@ -2263,7 +2263,7 @@ void nss_wifili_release_external_if(nss_
  */
 uint8_t nss_wifili_thread_scheme_alloc(struct nss_ctx_instance *nss_ctx,
 				int32_t radio_ifnum,
-				uint32_t radio_priority);
+				enum nss_wifili_thread_scheme_priority radio_priority);
 
 /**
  * nss_wifili_thread_scheme_dealloc
